modules/importer/src/main/scala/Importer.scala

```diff
@@ -179,7 +179,8 @@ object Importer extends SafeApp {
         val filter        = config.uget(Legacy.Instrument.F2.Filter)
         val lyoutWheel    = config.uget(Legacy.Instrument.F2.LyotWheel)
         val disperser     = config.uget(Legacy.Instrument.F2.Disperser)
-        F2Config(fpu, mosPreimaging, exposureTime, filter, lyoutWheel, disperser)
+        val windowCover   = config.uget(Legacy.Instrument.F2.WindowCover)
+        F2Config(fpu, mosPreimaging, exposureTime, filter, lyoutWheel, disperser, windowCover)
```

Comment:
`F2Config` is the Scala case class that defines the F2 configuration at a single step.  Exactly how this gets tied into the various step types though isn't clear yet.